dev/com.ibm.ws.microprofile.reactive.messaging.kafka/resources/com/ibm/ws/microprofile/reactive/messaging/kafka/resources/ReactiveMessaging.nlsprops

```diff
@@ -82,6 +82,11 @@ kafka.create.outgoing.retry.CWMRX1010W=CWMRX1010W: A Kafka outgoing connector fo
 kafka.create.outgoing.retry.CWMRX1010W.explanation=An error occurred when the Kafka connector was set up for an outgoing reactive messaging channel. This might be caused by a problem with the connector configuration, or by a problem resolving the Kafka broker hostname.
 kafka.create.outgoing.retry.CWMRX1010W.useraction=Review the error message, the messages.log file on the server, and FFDC logs to identify whether the problem was transient and was resolved by retrying the connector setup.
 
+# {0} = kafka record data, {1} = exception details
+# 'nack' is the name of a method and should not be translated 
+kafka.input.message.nacked.CWMRX1011E=CWMRX1011E: The nack method was called on a message created from a Kafka record, indicating that it was not be processed successfully. The reactive message stream will be shut down. The record was: {0}. The exception was: {1}.
```

Comment:
Extraneous word, 'be', in the statement, "indicating that it was not be processed successfully".  Please remove.